src/Forms/FormItem.js

```diff
@@ -33,11 +34,13 @@ FormItem.propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
     disableStyles: PropTypes.bool,
+    isHorizontal: PropTypes.bool,
```

Comment:
I don't love the name of this prop 🤷‍♀

Reply:
It looks like `isHorizontal` puts the label and the input side-by-side on the same visual row.  That seems more like `isInline` to me, but I also see that prop existed before.  I'm not quite sure what the difference is between `isHorizontal` and `isInline`.  Looking at the `fundamental-styles` page on forms, there are no uses of `fd-form-item--inline`. 🤷‍♂

Reply:
`--inline` is only for inline labels for radio and checkbox. I think we can just go forward like this, then with the rc that changes how checkboxes and radios work can revisit that?